src/containers/DebtPoolContainer/DebtPoolContainer.tsx

```diff
@@ -10,12 +10,11 @@ export interface Data {
   value: number
```

Comment:
Uwaga do całości DebtPoolContainer: w domyśle folder containerrs jest miejscem, gdzie wszystko podłączamy do Reduxa, więc przenieś może tu podłączanie selektora, bo to, co jest aktualnie, zostało tak tylko tymczasowo; zrób jeszcze analogiczny kontener dla StatsCollateral i złóż to w całość na StatisticsPage